apps/dashboard/src/components/footer/socialLinks.tsx

```diff
@@ -22,7 +22,7 @@ export const SOCIALS: socialLinkInfo[] = [
     ariaLabel: "Twitter",
   },
   {
-    icon: <SiDiscord fontSize={socialIconSize} />,
+    icon: <DiscordIcon fontSize={socialIconSize} className="size-5" />,
```

Comment:
remove fontSize

Reply:
ah thanks. Do you want me to remove fontSize on all of the icons in this file also?

Reply:
yes

Reply:
ok I removed it from the Discord icon. For the rest I think I should remove it from top of the stack, to avoid conflicts :D